app/src/ui/add-repository/create-repository.tsx

```diff
@@ -416,7 +416,7 @@ export class CreateRepository extends React.Component<
   private renderGitRepositoryWarning() {
     const isRepo = this.state.isRepository
 
-    if (this.state.path.length && !isRepo) {
+    if ((this.state.path.length && !isRepo) || this.state.path.length === 0) {
```

Comment:
Thanks for the contribution. This seems to complicate the if condition somewhat. Why not something this below?
```typescript
if (this.state.path.length === 0 || !isRepo) {
```

Reply:
I completely didn't catch the string type not being nullable. I'll update the PR 👍

Reply:
I've updated the PR. Good catch on the non-nullable string. It basically removes any possible edge cases.

Reply:
> It basically removes any possible edge cases.

Typescript will still let you assign `undefined` to a string and will accept `undefined` being passed as an argument to a function.

```ts
function f(a: string) {
  return a.length
}

f(undefined) //Cannot read property 'length' of undefined​​
```

☝️ TSC is happy to **not** call this an error and let it slide.

Reply:
Ah! okay - I'll implement a check for that case this evening 👍